src/main/java/com/autonomic/tmc/auth/exception/BaseSdkException.java

```diff
@@ -55,30 +55,35 @@ static String buildMessage(ErrorSourceType errorSourceType, String clientMessage
     static class ProjectProperties {
 
         private static final String UNKNOWN = "[ UNKNOWN ]";
-        static String pom = "pom.xml";
+        private Manifest manifest;
 
-        private Model properties;
-
-        public ProjectProperties() {
-            try {
-                MavenXpp3Reader reader = new MavenXpp3Reader();
-                properties = reader.read(new FileReader(pom));
-            } catch (IOException | XmlPullParserException e) {
-                log.warn("Failed to read pom.xml while building BaseSdkException", e);
-                properties = new Model();
+        private Manifest getManifest() {
+            if (Objects.isNull(manifest)) {
+                try {
+                    String jarPath = BaseSdkException.class.getProtectionDomain()
+                        .getCodeSource().getLocation().toURI().getPath();
+                    try (JarFile jarFile = new JarFile(jarPath)) {
+                        manifest = jarFile.getManifest();
+                    }
+                } catch (IOException | URISyntaxException ignored) {}
```

Comment:
```suggestion
                } catch (IOException | URISyntaxException ignored) {
                      log.warn("Unable to find Manifest to determine auth version", ignored);
                }
```
Since we already have `@Slf4j` , we might as well use it to get rid of Sonar error about ignoring exception without logging.